### PR TITLE
Adding proxy-from-gh sample

### PR DESCRIPTION
**Pull request by shanakama**
## Purpose
$subject

Related issue : https://github.com/wso2-enterprise/choreo/issues/30924

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: proxy-from-gh/pet-service/.choreo/component.yml
Comment (chameerar) [inline proxy-from-gh/pet-service/.choreo/component.yml:1]: This should be `.yaml` AFAIK. Please check

File: .azure/scripts/metadata_validator.py [hunk 1]
```diff
@@ -10,12 +10,13 @@
 VALID_COMPONENT_TYPES = [
     "service", "webhook", "manual-task", "scheduled-task", 
     "event-triggered", "event-handler", "test-runner", "many",
-    "web-application"
+    "web-application", "git-proxy"
```
Comment (chameerar) [inline .azure/scripts/metadata_validator.py:13]: Please add this to the `componentTypeMap` in choreo-console.
https://github.com/wso2-enterprise/choreo-console/blob/7351666de3727a38fda71d6a551bdb4a20b7f758/src/hooks/samples.tsx#L56

File: .azure/scripts/metadata_validator.py [hunk 2]
```diff
@@ -10,12 +10,13 @@
...
 ]
 
 VALID_BUILD_PACKS = [
     "ballerina", "wso2-mi", "go", "java", "php", "python", "nodejs", "ruby",
-    "vue", "many", "postman", "react", "docker", "angular", "dotnet", "prism-mock"
+    "vue", "many", "postman", "react", "docker", "angular", "dotnet", "prism-mock",
+    "none"
```
Comment (chameerar) [inline .azure/scripts/metadata_validator.py:19]: Please add this to the `buildPackNameMap` in choreo-console.
https://github.com/wso2-enterprise/choreo-console/blob/7351666de3727a38fda71d6a551bdb4a20b7f758/src/hooks/samples.tsx#L38C9-L38C25

**Review by chameerar — CHANGES_REQUESTED** [review]

File: .azure/scripts/metadata_validator.py (hunk 1 above)
Comment (shanakama) [reply .azure/scripts/metadata_validator.py:13]: Addressed : https://github.com/wso2-enterprise/choreo-console/pull/9704

File: .azure/scripts/metadata_validator.py (hunk 2 above)
Comment (shanakama) [reply .azure/scripts/metadata_validator.py:19]: Addressed : https://github.com/wso2-enterprise/choreo-console/pull/9704

File: proxy-from-gh/pet-service/swagger.yml [hunk 3]
```diff
@@ -0,0 +1,451 @@
...
+          type: string
+      xml:
+        name: '##default'
+  requestBodies:
+    Pet:
+      description: Pet object that needs to be added to the store
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Pet'
```
Comment (chameerar) [inline proxy-from-gh/pet-service/swagger.yml:451]: Check the new line here

**Review by chameerar — APPROVED** [review]

**Pull request merged**